File: assignment 3/armstrong.cpp
#include<iostream>
using namespace std;
class Armstrong
{
    int num,n,rem,res=0;
    public : void display()
    {
        cout<<"enter a number:\n";
        cin>>num;
        n=num;
        while(n!=0)
        {
            rem=n%10;
            res += rem*rem*rem;
            n /=10;
        }
        if(res==num)
        {
            cout<<"\nit is a armstrong number";
        }
        else
        {
            cout<<"\nit is not a armstrong number";
        }
    }
};
int main()
{
    Armstrong a1;
    a1.display();
}

File: JAVAAssignment 3/Armstrong.java
import java.util.Scanner;
class Armstrong
{
public static void main(String[] args)
{
int num,n,rem,res=0;
System.out.println("Enter a number");
Scanner sc=new Scanner(System.in);
num=sc.nextInt();
n=num;
while(n!=num)
{
rem=n%10;
res+=rem*rem*rem;
n/=10;
}
if(res==num)
{
System.out.println("It is a armstrong number");
}
else
{
System.out.println("It is not a armstrong number");
}
}
}